tracing/src/main/java/com/palantir/remoting3/tracing/Tracers.java

```diff
@@ -36,7 +37,24 @@ public static String randomId() {
     }
 
     static String longToPaddedHex(long number) {
```

Comment:
worth adding simple docs now that the implementation is more cryptic than `padStart(toHexString)`